services/ChartJsUtils.cfc

```diff
@@ -159,7 +166,17 @@ component {
 		// Apply chart-type-specific options
 		config = chart.applyTypeConfig( config );
 
-		return SerializeJson( config );
+		var serializedConfig = SerializeJson( config );
+
+		serializedConfig = replace( serializedConfig, '"PLUGINS_PLACEHOLDER"', plugins );
+
+		var rawScripts = chart.getRawScripts();
+
+		for ( var id in rawScripts ) {
```

Comment:
Its not clear to me what is happening here and how this is used. Can you update the documentation in the README.md file with a guide as to how a developer would use this?